web/pages/api/contact.js

```diff
@@ -96,6 +127,31 @@ const saveContactToDynamoDb = async (contact) => {
   }
 };
 
+const addToMailchimpList = async ({
+  email,
+  first_name,
+  last_name,
+  phone,
+  organization,
+}) => {
+  try {
+    await mailchimpClient.lists.addListMember(gatedDocumentListId, {
```

Comment:
Just double checking; there's no time we would get a successful response if there was an unexpected behavior, right? If they are already subscribed they would get a 200 response? Or is there value in parsing the response to give the user more info?

Reply:
Right now if the email is already a member of the list it returns a 400 response and kicks off the alert on the front end that only tells the user there was an error, but no specifics. Definitely worth getting more detail on that.

Reply:
Yeah, let's at least send some appropriate error messages back

Reply:
I have the alert popping up with the descriptive title from Mailchimps error response.